builds/misc/packages-release.yaml

```diff
@@ -987,14 +994,33 @@ stages:
                 secretsFilter: >-
                   GitHubAccessToken
             - checkout: self
+            - checkout: azure-iotedge
+              submodules: recursive
             - bash: |
                 echo "Approval Complete"
 
+                BASE_VERSION="$(cat $BUILD_SOURCESDIRECTORY/iotedge/edgelet/version.txt)"
+                VERSION="$BASE_VERSION"
+                AZURE_IOTEDGE_REPO_PATH="$(Build.SourcesDirectory)/azure-iotedge"
+                IOTEDGE_REPO_PATH="$(Build.SourcesDirectory)/iotedge"
```

Comment:
Same here. I thought `checkout: self` checks out into Build.SourcesDirectory but doesn't include the top-level repo directory. I think it only includes the top-level dir when you're checking out a different (i.e. non-self) repo such as what you're doing with azure-iotedge.

Reply:
Please see the comment above.